test/node/msw-api/many-request-handlers-jsdom.test.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * @jest-environment jsdom
+ */
+import { graphql, http, HttpResponse } from 'msw'
+import { setupServer } from 'msw/node'
+
+// Create a large number of request handlers.
+const restHandlers = new Array(100).fill(null).map((_, index) => {
+  return http.post(
+    `https://example.com/resource/${index}`,
+    async ({ request }) => {
+      const text = await request.text()
+      return HttpResponse.text(text + index.toString())
+    },
+  )
+})
+
+const graphqlHanlers = new Array(100).fill(null).map((_, index) => {
+  return graphql.query(`Get${index}`, () => {
+    return HttpResponse.json({ data: { index } })
+  })
+})
+
+const server = setupServer(...restHandlers, ...graphqlHanlers)
+
+beforeAll(() => {
+  server.listen()
+  jest.spyOn(process.stderr, 'write')
+})
+
+afterAll(() => {
+  server.close()
+  jest.restoreAllMocks()
+})
+
+it('does not print a memory leak warning when having many request handlers', async () => {
+  const httpResponse = await fetch('https://example.com/resource/42', {
```

Comment:
This test currently throws an error "ReferenceError: fetch is not defined". I did not find a reference for JSDOM and calling fetch in an integration test. How would you like to handle this?

Reply:
I will look into this.

Reply:
Fixed by https://github.com/mswjs/msw/pull/1779/commits/5ae47068a6553a680c08a4de17e3a0b6c1515abd. I will never get tired of recommending people migrate away from Jest :(